test-dxcompiler-in-project/Readme.md

```diff
@@ -0,0 +1,21 @@
+# test-dxcompiler-in-project Readme
+
+Applet `test-dxcompiler-in-project` can be built and deployed in customer-provided projects to facilitate testing on their WDL workflows.
+
+## Test Project Setup
+
+- The [example_project directory](./example_project) shows the recommended structure for a test project.
+- `test/test_wdl.sh` should be a script that downloads the WDL files onto the worker, compiles the workflows, and runs them.
+- It is recommended to have 1 `workflow_#` folder per test workflow containing the WDL and auxiliary files necessary to compile the workflow.
+- It is recommended to direct compiled artifacts and workflow outputs to `test_out/workflow_#`.
+- Input data files should be kept elsewhere; they do not need to be downloaded for compilation.
+
+## Test Project Access
+
+- Customer projects should be shared with `org-dnanexus_apps_customer_testers`.
+
+## Building & Running the Applet
+
+- Upload regional dxCompiler assets to the platform and stage dxCompiler.jar in the root `dxCompiler` folder. This can be done by running the initial stage of a release.
```

Comment:
@jdidion @emiloslavsky   This process assumes we would be doing the testing after uploading some release candidate artefacts to the platform, but before publishing anything to GitHub / Maven. This is fine if one of us is running these tests as part of a release, but if we hand over the process to the support team, we might want to publish an RC jar to GitHub and fetch it from there during `make`.